net/pfSense-pkg-WireGuard/Makefile

```diff
@@ -0,0 +1,79 @@
+# $FreeBSD$
+
```

Comment:
These tags were removed all over FreeBSD ports tree since it moved from svn to git.  Please remove them